ereolen_support/ereolen_support.info

```diff
@@ -0,0 +1,6 @@
+name = eReolen support
+description = Support form for eReolen
```

Comment:
Maybe `Support form for eReolen that integrates with Jira`

Reply:
Why the specificity. We also support Publizon mail, and others may come as well.